File: sdk/twig/TwigSafeString.php
<?php
namespace twig;

/**
 * Value for raw text in templates.
 * @package twig
 */
class TwigSafeString
{
    /**
     * TwigSafeString constructor.
     * @param string $text
     */
    public function __construct(string $text)
    {
    }

    /**
     * Returns a text of safe string
     * @return string
     */
    public function __toString(): string
    {
    }
}

File: src-jvm/main/java/php/pkg/twig/classes/TwigSafeString.java
package php.pkg.twig.classes;

import com.mitchellbosecke.pebble.extension.escaper.SafeString;
import php.pkg.twig.TwigExtension;
import php.runtime.annotation.Reflection.Namespace;
import php.runtime.annotation.Reflection.Signature;
import php.runtime.env.Environment;
import php.runtime.lang.BaseWrapper;
import php.runtime.reflection.ClassEntity;

@Namespace(TwigExtension.NS)
public class TwigSafeString extends BaseWrapper<SafeString> {
    public TwigSafeString(Environment env, SafeString wrappedObject) {
        super(env, wrappedObject);
    }

    public TwigSafeString(Environment env, ClassEntity clazz) {
        super(env, clazz);
    }

    @Signature
    public void __construct(String str) {
        __wrappedObject = new SafeString(str);
    }

    @Signature
    public String __toString() {
        return getWrappedObject().toString();
    }
}
